openslides/motions/views.py

```diff
@@ -152,8 +152,8 @@ def update(self, request, *args, **kwargs):
 
         # Check permissions.
         if (not has_perm(request.user, 'motions.can_manage') and
-            not (motion.is_submitter(request.user) and
-                 motion.state.allow_submitter_edit)):
+                not (motion.is_submitter(request.user) and motion.state.allow_submitter_edit) and
+                not has_perm(request.user, 'motions.can_see_and_manage_comments')):
             self.permission_denied(request)
 
         # Check permission to send only some data.
```

Comment:
This is not correct I think. Someone who has only the permission to see and manage comments can now update title, text and reason for any motion.